docs/quickstart.md

```diff
@@ -2,78 +2,30 @@
 
 ## Pre-reqs
 
-The below instructions assume Argo CloudOps is on your local OSX system
-with Docker Desktop managing resource in AWS (region us-west-2) with credentials provided by Vault.
-
 * Install [Docker Desktop](https://www.docker.com/products/docker-desktop), ensure kubernetes is running.
 
 * Install [Argo Workflows](https://argoproj.github.io/argo-workflows/installation/)
 
 * Install **Argo CLI** `brew install argo`
 
-* Install **GoLang** `brew install golang`
-
-* Install GoLint `go get -u golang.org/x/lint/golint` and ensure `$GOPATH` is in your `$PATH`.
-
 * Install [PostgreSQL](https://www.postgresql.org/download/)
 
-* Submit Argo Hello World workflow and record the **Name** from the output.
-
-    ```
-    argo submit -n argo https://raw.githubusercontent.com/argoproj/argo-workflows/master/examples/hello-world.yaml
-    ```
-
-* Ensure the workflow completes with Status **Succeeded**.
-
-    ```
-    argo get -n argo <UPDATE_WITH_NAME_FROM_ABOVE> |grep Status
-    ```
-
 * Install [Vault](https://www.vaultproject.io/downloads) for credential generation.
 
 * Install [jq](https://stedolan.github.io/jq/) for json parsing.
 
-* Install **npm** `brew install npm` (For CDK).
-
-* Install [terraform](https://www.terraform.io/downloads.html).
-
 ## Deploy Sample App Locally
 
 You will need two windows
 
 1. Vault & Argo CloudOps Service
 1. Client commands, etc
 
-### One Time Setup
-
-* In window **#1**, ensure you have AWS credentials for the target account.
-
-* Create the IAM role which will be used for the sample project.
-
-    ```sh
-    bash scripts/create_iam_role.sh
-    ```
-
-* Create a new postgres database. This can be done using the command:
-
-    ```sh
-    createdb argocloudops
```

Comment:
It looks like this is still needed, also I had to use `createdb argocloudops -U postgres`

Reply:
The script includes the creation, but i'm guessing it failed since it needed -U postgres, updating